skbeam/core/correlation.py

```diff
@@ -121,7 +119,7 @@ def _one_time_process(buf, G, past_intensity_norm, future_intensity_norm,
 
 results = namedtuple(
     'correlation_results',
-    ['g2', 'lag_steps', 'internal_state']
+    ['correlation_results', 'lag_steps', 'internal_state']
```

Comment:
Usually `g2` is for one-time, but somie papers they call two-time `g2`. I don't find the conventional name for the results from two-time correlation. Anyway, Tomorrow I will discuss this with Andrei and Yugang.